src/cc2olx/models.py

```diff
@@ -2,6 +2,7 @@
 import re
 from textwrap import dedent
 import zipfile
+import imghdr
```

Comment:
I like to order the imports alphabetically. That way it's easier to check if one exists. Some people at EdX like to do that also.

Reply:
I have updated the import statement and put it above import re.